xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.cpp

```diff
@@ -81,6 +96,7 @@ typedef struct
 } dxva2_mode_t;
 
 /* XXX Preferred modes must come first */
+// clang-format off
```

Comment:
Is this really needed? You can split the formatting of the following code by preformatting it with clang-format using specific lines only. Then you can commit that and make your changes on top to have a clean commit

```
clang-format -i --lines=100:133 xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.cpp
```

Reply:
Seems more readable like this and is allowed in Code Guidelines point "3.7. Exceptions to the Formatting Rules For Better Readability"